README.rst

```diff
@@ -178,23 +165,10 @@ you can receive all the latest updates, and switch back to version 5.x behavior:
     >>> dt.tzinfo
     <DstTzInfo 'Europe/Vienna' CET+1:00:00 STD>
 
-Version 6 is on `branch main <https://github.com/collective/icalendar/>`_ with compatibility to Python versions ``3.7+`` and ``PyPy3``.
+Version 6 is on `branch main <https://github.com/collective/icalendar/>`_.
+It is compatible with Python versions 3.7 - 3.11, and PyPy3.
```

Comment:
I just noticed that the issue name says 3.8+, but the trove classifiers and this line say 3.7-3.11. Which is correct?

Reply:
For version 6, 3.8+ is correct. #634 is included in #630.

Reply:
OK, then we will need to update #634 to include `CHANGES.rst` with the correct Python versions.

Reply:
But wait a minute, look at the change log entry, the issue title, and this line of test. See the discrepancy in versions? They don't align with each other.

Reply:
I know. Please change it as you like it. I would like to be consistent between releases, not always. There are other PRs changing this at the same time.
It can be changed after merge, too. If you think, this is harmful, then, do not merge it. Otherwise I prefer progress over perfection.

Reply:
I'll correct it in a review suggestion. I prefer to get details right as soon as I realize them, else I will likely forget to deal with them later.

Reply:
Side note: I assume I cannot push to your remote, but I have not tried. For ease of collaboration where others can push to this remote (`collective/icalendar`), consider using this remote when you push.

Reply:
Ok. This is a good suggestion to use this for collaboration. I sometimes create pull requests to other people's branches. I did not want to clutter what is going on here. Maybe, if I name that branches issue-..., then it is not so much of a clutter.